sentry/templates/configmap-snuba.yaml

```diff
@@ -11,36 +11,28 @@ metadata:
 data:
   settings.py: |
     import os
-    from typing import Any, Mapping, Sequence
+
+    from snuba.settings import *
 
     env = os.environ.get
 
     DEBUG = env("DEBUG", "0").lower() in ("1", "true")
 
-    DEFAULT_BROKERS = [{{ printf "%s:%s" (include "sentry.kafka.host" .) (include "sentry.kafka.port" .) | quote }}]
+    # Clickhouse Options
+    CLUSTERS[0]["host"] = env("CLICKHOUSE_HOST", {{ include "sentry.clickhouse.host" . | quote }})
+    CLUSTERS[0]["port"] = int(env("CLICKHOUSE_PORT", {{ include "sentry.clickhouse.port" . }}))
+    # FIXME: Snuba will be able to migrate multi node clusters in the future
+    CLUSTERS[0]["single_node"] = env("CLICKHOUSE_SINGLE_NODE", "false").lower() == "true"
 
+    # Redis Options
     REDIS_HOST = {{ include "sentry.redis.host" . | quote }}
     REDIS_PORT = {{ include "sentry.redis.port" . }}
     {{- if $redisPass }}
     REDIS_PASSWORD = {{ $redisPass | quote }}
     {{- end }}
     REDIS_DB = int(env("REDIS_DB", 1))
-    USE_REDIS_CLUSTER = False
```

Comment:
You removed this and both DOGSTATSD, I just want to make sure it's on purpose?

Reply:
Yes, we should now be able to inherit the values from the default `settings.py`

Reply:
👍